components/Embed.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface Props {
+  src: string;
+  title: string;
```

Comment:
**praise:** Anything besides src should be optional and have default params. Title empty, width 100 %, height I think 300px.